analyzer/runtime/runtime.go

```diff
@@ -177,7 +183,8 @@ func (m *Main) Start() {
 
 // Name returns the name of the Main.
 func (m *Main) Name() string {
-	return EmeraldDamaskAnalyzerName
+	panic("we never actually call Name")
```

Comment:
:)
Let's just get rid of it at the interface level then

Reply:
Edit: We call it [here](https://github.com/oasisprotocol/oasis-indexer/blob/699e4cccbde673c2e61bbc25822401512e756db4/cmd/analyzer/analyzer.go#L156-L156)
But it's true that from I can tell, we never truly use that return value. We do use it as a map key, but then only ever iterate over the values of the map and ignore the key. So if we change the map to a list, we're good to go.

Reply:
:O

Reply:
keeping then. gonna remove the panic

Reply:
removed